### ISSUE TITLE
Accessibility: Hamburger menu and Talkback

### ISSUE THREAD

**Issue by davidbtc2009** [TylerWilliamson/QuickWeather#42]
When the hamburger menu is closed talkback reads "Drawer Opened, Button" and when the menu is open it reads "Drawer Closed, Button" this is quite confusing. Maybe the menu button should have the label "menu" and the back arrow coukd be "close menu"

**Comment by TylerWilliamson** [comment]
Oops. I thought the labels were for the state of the drawer, not the action being completed. I'll change it so that the button says "Open Drawer" when the drawer is closed, and "Close Drawer" when it is open. Are there any other accessibility issues that you have encountered?

**Comment by davidbtc2009** [comment]
So far just one other. For each of the 7 day forcasts talkback says "Forecast Date, moderate rain, Daily High Temperature, Daily Low Temperature, Forecast". Another example is "Forecast Date, light rain, Daily High Temperature, Daily Low Temperature, Forecast" instead of something like "saturday moderate rain high 89 low 71"

**Comment by TylerWilliamson** [comment]
Ah. I'll look into those as well and do some additional TalkBack testing

**Comment by davidbtc2009** [comment]
Thanks, fantastic app so far

**Comment by davidbtc2009** [comment]
I found some other issues but opened a seperate issue since its not talkback related. https://github.com/TylerWilliamson/QuickWeather/issues/44

**Comment by TylerWilliamson** [comment]
Thanks for finding that. I wanted to update you and let you know that I've been doing a lot of TalkBack development and testing. Should be able to include it in the next release. The Talkback should be more intuitive and read the weather information better as well as better informing users about the actions available on screen.

I'll also look into the other accessibility issues as well.

**Comment by TylerWilliamson** [comment]
Resolved in [2.2.3](https://github.com/TylerWilliamson/QuickWeather/releases/tag/2.2.3)
